perfmetrics/scripts/run_load_test_and_fetch_metrics.sh

```diff
@@ -30,21 +30,27 @@ git checkout c5d8ce3fc736210ded83b126c71e3225c7ffd7c9 && \
 ./configure && make && sudo make install
 
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
+
```

Comment:
There were some conflicts here with the changes in @Tulsishah's [commit](https://github.com/GoogleCloudPlatform/gcsfuse/commit/1a5db44f520d4854d5418342ca597a85f654f5c8#diff-ff86a10afec13e1928d1cf8ff4f5c5c5fe7aed9794cd54e6931d9509e1b96643) in master and @ashmeenkaur's [commit in nov_2023_release](https://github.com/GoogleCloudPlatform/gcsfuse/commit/42c828c1e8278d84211aa28908eb9cb3403ec9fd). Please check this file carefully.

Reply:
Looks good to me.

Reply:
@raj-prince thanks.